### PR TITLE
Implement GetAddressOfOp

### PR DESCRIPTION
**Pull request by marbre**

### CODE REVIEW

**Review by simon-camp — APPROVED** [review]
LGTM besides the comment on spacing.

**Review by simon-camp — CHANGES_REQUESTED** [review]
I think the printer can be simplified. Otherwise LGTM.

File: lib/Target/Cpp/TranslateToCpp.cpp [hunk 1]
```diff
@@ -109,6 +109,20 @@ static LogicalResult printCallOp(CppEmitter &emitter, emitc::CallOp callOp) {
...
 
+static LogicalResult printGetAddressOfOp(CppEmitter &emitter,
+                                         emitc::GetAddressOfOp getAddressOfOp) {
+  auto &os = emitter.ostream();
+
+  auto outputType = getAddressOfOp.result().getType();
+  if (failed(emitter.emitType(outputType)))
+    return failure();
+  os << " ";
+  os << emitter.getOrCreateName(getAddressOfOp.getResult()) << " = &";
```
[the remark this answers is not in the document]
Comment (marbre) [reply lib/Target/Cpp/TranslateToCpp.cpp:120]: Good catch.

[the remark this answers is not in the document]
Comment (marbre) [reply lib/Target/Cpp/TranslateToCpp.cpp]: Done.

**Review by simon-camp — APPROVED** [review]
Merge when the lights go green :)

**Pull request merged**